src/text/raw_text_writer.rs

```diff
@@ -59,43 +68,44 @@ impl RawTextWriterBuilder {
     /// ```
     pub fn pretty() -> RawTextWriterBuilder {
         RawTextWriterBuilder {
-            // Each value appears on its own line
-            space_between_values: String::from("\n"),
-            // Values get four spaces of indentation per level of depth
-            indentation: String::from("    "),
-            // Field names and values are separated by a single space
-            space_after_field_name: String::from(" "),
-            // The first value in a container appears on a line by itself
-            space_after_container_start: String::from("\n"),
+            whitespace_config: WhitespaceConfig { ..PRETTY_WS_CONFIG },
         }
     }
 
-    pub fn with_space_between_values<S: Into<String>>(
+    pub fn with_space_between_top_values(
```

Comment:
For consistency with other usages, I'd like to use the wordier `with_space_between_top_level_values`. (Note the **\_level\_**)

Reply:
Done!